exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
+let message = 'Its cold outside!'
```

Comment:
You have a spelling mistake here, instead of Its use It's please.When iI rush I do the same mistake :)